pontoon/administration/files.py

```diff
@@ -129,21 +131,18 @@ def modify_entity_mine(self, id, value):
 """ End monkeypatching """
 
 
-def get_locale_paths(project, locale):
+def get_locale_paths(project, locale, source_paths, source_directory):
```

Comment:
That way we only extract locale paths that have source paths counterparts available. This is necessary for this PR, since we use per-project en-US repos (to only display e.g. Firefox strings in Firefox project) and full locale repos, which contain files for all (up to 5, depending on locale) Mozilla projects.